pkg/cli/cmd/radinit/display_test.go

```diff
@@ -66,12 +54,18 @@ func Test_summaryModel(t *testing.T) {
 
 		waitForRender(t, tm.Output())
 
-		// Press ENTER
-		tm.Send(tea.KeyMsg{Type: key})
+		// Press the given key
+		tm.Send(tea.KeyMsg{
+			Type: key,
+		})
 
-		// Wait for final render and exit.
-		tm.WaitFinished(t, teatest.WithFinalTimeout(waitTimeout))
-		waitForEmpty(t, tm.FinalOutput(t))
+		if err := tm.Quit(); err != nil {
```

Comment:
Please see example tests from the original repo: https://github.com/charmbracelet/x/blob/20117e9c8cd5ad229645f1bca3422b7e4110c96c/exp/teatest/app_test.go#L34. We can discuss about them.

Reply:
This is a different test though right? We're now explicitly telling the model to quit.